js/src/personal/haoking/Haoking.aboutme.tsx

```diff
@@ -0,0 +1,35 @@
+import { MantineProvider, Box, Flex, rem } from '@mantine/core'
+import { LeftAboutMe } from './LeftAboutMe.tsx'
+import { RightAboutMe } from './RightAboutMe.tsx'
+import { Navbar } from './Navbar.tsx'
+import { theme } from './theme.ts'
+
+export function HaokingAboutMe({
```

Comment:
Just name this AboutMe.tsx or AboutMe.page.tsx
I think we want to have .page.tsx for top level pages that are hit by the react router.

Reply:
Renamed to AboutMe.tsx